src/components/Features/Bridge/Bridge.slice.js

```diff
@@ -3,15 +3,19 @@ import {createSlice} from '@reduxjs/toolkit';
 import {MenuType} from '../../../enums';
 
 const initialState = {
-  menu: MenuType.TRANSFER
+  menu: MenuType.TRANSFER,
+  menuProps: {
+    transferId: null
+  }
```

Comment:
Lets maybe change to:
`
  menuProps: {
    [MenuType.ACCOUNT]: {
      transferId: null
    },
  }
`